README.md

```diff
@@ -3,91 +3,70 @@
 
 ## Requirements
 
-- Python 3.x
-- Postgres >= 9
+- [Docker](https://www.docker.com/)
+
 
 ## Running the app locally
 
-Perform the following steps from your terminal.
+We use Docker for local development. To get started, run the following from your terminal:
 
 1. Clone this repository and `cd` into your local copy.
 
     ```bash
     git clone git@github.com:datamade/committee-oversight.git
     cd committee-oversight
     ```
-2. Create a virtual environment. (We recommend using [`virtualenvwrapper`](http://virtualenvwrapper.readthedocs.org/en/latest/install.html) for working in a virtualized development environment.)
 
-    ```bash
-    mkvirtualenv committee-oversight
-    ```
-3. Install the requirements.
+2. Load initial category data:
 
     ```bash
-    pip install -r requirements.txt
+    docker-compose -f docker-compose.yml -f docker-compose.db-ops.yml run --rm dbload
```

Comment:
This step is included because I thought I'd have to preserve the step of individually loading category names. I was wrong, I think. As that will be included in the current `hearings.dump` and any future data backup, we can safely skip this step entirely. I've removed it in the latest commit—could you give the steps in the README another shot?